test/trivial/diten/printVisibleConflict.good

```diff
@@ -0,0 +1,14 @@
+  <internal>:0: _local
+  <internal>:0: _privatization
+  <internal>:0: boundsChecking
+  <internal>:0: castChecking
+  <internal>:0: chpl_checkDivByZero
+  <internal>:0: chpl_checkNilDereferences
+  <internal>:0: chpl_nodeID
+  <internal>:0: chpl_overloadSetsChecks
+  <internal>:0: chpl_warnUnstable
+  modules/minimal/internal/ChapelLocale.chpl:27: chpl_sublocID_t
+  printVisibleConflict.chpl:11: xyzSymbol
+printVisibleConflict.chpl:12: Printing symbols visible from here:
+printVisibleConflict.chpl:2: error: symbol zzzSymbol is multiply defined
```

Comment:
No, you're right. That's a surprise to me. I wouldn't have expected that we allow that situation at all, but it's only if you actually try to use the multiply-defined symbol that we issue the error.

Reply:
I think it's okay, since we can get into situations where multiple modules use the same name, but you can still distinguish which one you mean via qualified access (e.g., even if `foo` doesn't work because A and B define a foo, you can still write `A.foo` or `B.foo` to resolve the issue)

Reply:
I added a check to avoid issuing the error in this case.

Reply:
Ah, so ready for a second look?

Reply:
Looks like you have a conflict to resolve, too